web/service-worker.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-restricted-globals */
+self.addEventListener('fetch', (event) => {
+  // Ignore
```

Comment:
So, can we not allow "installing" the app without this totally pointless event listener?  Seems weird to have to have something that does nothing.  If we are just trying to please the Lighthouse test I think we can skip it.  I guess I need to look into it more.

Reply:
Also still unclear on if there is any point or advantage to having it cache our main, compiled `wv.js` file or not.  If there is, I suppose that could be something to do here.

Reply:
> So, can we not allow "installing" the app without this totally pointless event listener? Seems weird to have to have something that does nothing. If we are just trying to please the Lighthouse test I think we can skip it. I guess I need to look into it more.

When I tested locally, I removed the `<script>` tag from the index.html that's calls the service worker file, but left the manifest `<link...>` tag in there to see if we only need the manifest file.  I was not able to install on desktop nor did I get a splash screen or full screen app.  Reenabled the service worker and everything worked again.  Chrome/chromium browsers seem to expect a service worker of some sort, even if it's doing nothing.

Here are a couple of pages regarding minimal PWA requirements.  Essentially what we're doing here is a "Network-Only" approaching to caching.  In other words, we're not caching at all, but by declaring that we are making a decision on how we want to cache things.
- https://mobiforge.com/design-development/pwa-minimus-a-minimal-pwa-checklist
- https://stackoverflow.com/questions/61208907/pwa-minimal-service-worker-just-for-trigger-the-install-button
- https://developers.google.com/web/ilt/pwa/caching-files-with-service-worker#network_only

> Also still unclear on if there is any point or advantage to having it cache our main, compiled `wv.js` file or not. If there is, I suppose that could be something to do here.

Def worth a shot I think.  If someone visits Worldview and/or installs the app, the `wv.js` file won't need to be downloaded again on subsequent visits or page reloads.

Reply:
More thoughts on this... I think it would be preferable not to attach an event listener to `fetch` events since this will  needlessly fire for every network request.  If we have to register for _some_ event perhaps the `install` event would be better since it would happen only once?

Reply:
Update:  It looks like this is actually one of the install criteria: `Registers a service worker with a fetch handler`
https://web.dev/install-criteria/

Reply:
Circling back to this -- So just to pass the install criteria, we should keep the service worker, as is?  At least for now?

Reply:
Yep no need for changes to this